docs/guides/replace-servers-url.md

```diff
@@ -6,41 +6,48 @@ This page describes how to replace the server URL with a decorator for a given e
 
 ## Prerequisites
 
-{% admonition type="note" name="We do, You do" %}
-This tutorial is most effective when you follow along and complete the steps.
-{% /admonition %}
+- [Install @redocly/cli](../installation.md). We use version 1.12.0 in this tutorial.
 
-- [Install @redocly/cli](../installation.md) with version 1.0.0-beta.111 or later (we use 1.0.0-beta.111 in this tutorial).
-- Save the following OpenAPI file as `original.yaml` into a new directory named `replace-servers-demo`.
-  ```yaml
-  openapi: 3.1.0
-  info:
-    version: 1.0.0
-    title: Custom decorators demo
-    description: The servers URL is replaced by the decorator during the `bundle` process.
-  servers:
-    - url: 'https://example.com/api/v1'
-  paths:
-    /status:
-      get:
-        summary: Get status
-        operationId: getStatus
-        security: []
-        responses:
-          '204':
-            description: Status OK
-          '400':
-            description: Status not OK
-  ```
 - Use your favorite IDE for editing files (we use VS Code and have the [Redocly extension](../../redocly-openapi/index.md) installed).
 
-## Step 1: Create a custom plugin
+## Create a demo specification file
+
+In this step, create an OpenAPI specification file.
```

Comment:
```suggestion
In this step, create an OpenAPI definition file.
```

Reply:
Maybe we should say here why users are creating this folder and file. Something like, before we can create the plugin we need a sample definition document to test it with. In this step we will create the sample folder and file to use for testing our plugin later.

Reply:
DESCRIPTION. Not definition. (sorry for shouting, caps lock was on but I figured it sort of made my point :laughing: )